openedx/features/pakx/lms/pakx_admin_app/serializers.py

```diff
@@ -15,11 +15,7 @@ class Meta:
         fields = ('id', 'email', 'name', 'employee_id', 'language', 'is_active', 'role')
 
     def get_role(self, obj):
-        if obj.is_superuser:
-            return ADMIN
-        elif obj.is_staff:
-            return STAFF
-        elif obj.groups.filter(name=GROUP_TRAINING_MANAGERS).exists():
-            return TRAINING_MANAGER
+        if obj.staff_groups:
+            return TRAINING_MANAGER if obj.staff_groups[0].name == GROUP_TRAINING_MANAGERS else ORG_ADMIN
```

Comment:
can we use something else instead of the hard coded index?

Reply:
Actually, we are using prefetch for groups (TM/Org_Admin) to avoid quires in the loop. Since `.first()` will execute 1 query for each obj while indexing will use already prefetched data. Plus we are also making sure that at this point obj will be either part of the TM or Org_Admin group.